sandboxes/README.md

```diff
@@ -0,0 +1,124 @@
+# Sandboxes for Analysis
+
+## Adding a new Runtime Analysis Sandbox
+
+Each runtime analysis sandbox requires:
+
+- a command to trigger the runtime analysis phases, usually named
+  `analyze.[ext]`.
+- a `Dockerfile` for constructing the sandbox.
+
+### Analysis Command
+
+Each command must conform to the following API specification.
+
+#### Arguments
+
+- Must take 2 required arguments and 1 optional argument:
+    1. phase name
+        - must support at least "all", "install".
+        - may also include "import" and any other phases useful to the
+          ecosystem.
+    1. package:
+        - either a local file contain a package to install;
+        - or the name of the package in the package repository to install.
```

Comment:
Should we add an explicit '--local' flag to make distinguishing between these two cases more explicit?

Reply:
Added. I was iffy about this, but I think being explicit is better than implicit.